services/galley/src/Galley/Data.hs

```diff
@@ -551,6 +554,19 @@ conversationIdsFrom usr start (fromRange -> max) =
   where
     strip p = p {result = take (fromIntegral max) (result p)}
 
+conversationIdsPageFrom ::
+  (MonadClient m, Log.MonadLogger m, MonadThrow m) =>
```

Comment:
Is the logger constraint needed? Maybe it is needed by `paginateWithState`, but I haven't checked myself.